### ISSUE TITLE
Implement a "dashboard" (Summary of Impact) to provide quick and easy access to directory data

### ISSUE THREAD

**Issue by dayotte** [platform-coop-toolkit/coop-map-directory-index#35]
This data should include the number of co-ops, sectors and people impacted, with details to be contained in a table or spreadsheet (probably also location)

**Comment by dayotte** [comment]
@erictheise something like this? Haven't thought through what the Details page would look like exactly but it sounded like what is wanted is a table or spreadsheet, I'm thinking with some simple sorting options (e.g. by name, sector, number impacted).
![Dashboard data](https://user-images.githubusercontent.com/6207009/81021652-7d451b00-8e20-11ea-8570-747249247ed5.png)

**Comment by cherylhjli** [comment]
Is this a different accordion than the map view summary? I don't have a view of what other data is beneath the accordion, I wonder if it's possible to make them the same thing?

**Comment by erictheise** [comment]
@dayotte, rather than _dashboard_ could we call this the _heartbeat_ (my words) or _ticker tape_ (Trebor's words) or some other name as this is quite a minimal summary. What I and I think Trebor envision for a _dashboard_ is more of a full screen spreadsheet/database view that a policy analyst could work with to calculate high level aggregate numbers.

**Comment by dayotte** [comment]
@erictheise yes, I'm imagining that selecting "Details" from the "ticker-tape"/banner would reveal the spreadsheet/database view

**Comment by dayotte** [comment]
> Is this a different accordion than the map view summary? I don't have a view of what other data is beneath the accordion, I wonder if it's possible to make them the same thing?

@cherylhjli yes, it is different from the map view summary.  It provides a summary of everything contained in the directory, and is limited to a few pieces of data (number of co-ops, sectors and people impacted). It would provide an entry point into the "dashboard" or database/spreadsheet that @erictheise described above. The map view summary would describe only what is contained in the visible map area, and would include more info about the types of listings etc.

**Comment by dayotte** [comment]
Suggest putting this on the About page. Need to confirm what content is in the "summary sheet". Imagining a sortable table listing all the co-op names along with the sector each one is in and number of people impacted by each.
![About page desktop](https://user-images.githubusercontent.com/6207009/82380322-1f4a3300-99dd-11ea-8783-3abb9cdb7c9d.png)
![About page mobile](https://user-images.githubusercontent.com/6207009/82380325-2113f680-99dd-11ea-9197-cbe88b5307ef.png)

**Comment by cherylhjli** [comment]
Looks good, I wonder if a more descriptive title might be helpful? Like "Summary of impact by co-ops" or something

**Comment by dayotte** [comment]
thanks for the suggestion @cherylhjli , updated mockups attached
![About page desktop](https://user-images.githubusercontent.com/6207009/82599411-15513d00-9b61-11ea-912c-b042731b9b54.png)
![About page mobile](https://user-images.githubusercontent.com/6207009/82599413-16826a00-9b61-11ea-829b-608ec2971704.png)

**Comment by dayotte** [comment]
@greatislander, when @cherylhjli and I talked through this we realised it would probably be best if the Summary of Impact had its own page if possible, and thought it could live under About, such that About would include 2 pages - About the Directory and Summary of Impact. We have also added the ability to filter by country. If too much for MVP let's discuss how to simplify. The design includes a link to "Download detailed impact summary" which would pull all the impact summary data into a spreadsheet, but we can leave this out for now. 

![Summary of Impact](https://user-images.githubusercontent.com/6207009/86155104-ccbe6500-bab8-11ea-9324-74df6ba1b173.png)
